consensus/scp/src/slot.rs

```diff
@@ -58,16 +58,16 @@ pub struct Slot<V: Value, ValidationError: Display> {
     M: HashMap<NodeID, Msg<V>>,
 
     /// Set of values that have been proposed, but not yet voted for.
-    W: BTreeSet<V>,
+    W: HashSet<V>,
```

Comment:
These are no longer ordered sets. The ordering of values in a Ballot is entirely determined by the combine function, which may be a function of data that is not present in V -- specifically, when V is TxHash, but the ordering depends on the contents of WellFormedTxContext.